### ISSUE TITLE
Creating standalone_squidex_proxy_1 ... error

### ISSUE THREAD

**Issue by Shadowman4205** [Squidex/squidex-docker#1]
Getting this error

```
ERROR: for standalone_squidex_proxy_1  Cannot start service squidex_proxy: driver failed programming external connectivity on endpoint standalone_squidex_proxy_1 (dd8ce05e1c2fe772393ea44d58aab4eafd1b641f8e7bd8872692da7feb7a4d79): Error starting userland proxy: Bind for 0.0.0.0:80: unexpected error Permission denied

ERROR: for squidex_proxy  Cannot start service squidex_proxy: driver failed programming external connectivity on endpoint standalone_squidex_proxy_1 (dd8ce05e1c2fe772393ea44d58aab4eafd1b641f8e7bd8872692da7feb7a4d79): Error starting userland proxy: Bind for 0.0.0.0:80: unexpected error Permission denied
ERROR: Encountered errors while bringing up the project.
```

**Comment by SebastianStehle** [comment]
I am not sure if this is really a Squidex error? Can you provide more information? Is the port 80 already in sue by the host machine?

**Comment by Shadowman4205** [comment]
ok i solve this error by going inside docker-compose and change port for 80. It was error because the port 80 is being used by anothter app.

**Comment by Shadowman4205** [comment]
@SebastianStehle  

how i access the app now

after doing this 

```
...TempProjects\squidex-docker\standalone>docker-compose up -d
Starting standalone_squidex_mongo_1 ...
Starting standalone_squidex_mongo_1 ... done
Creating standalone_squidex_squidex_1 ...
Creating standalone_squidex_squidex_1 ... done
Creating standalone_squidex_proxy_1 ...
Creating standalone_squidex_proxy_1 ... done
Creating standalone_squidex_encrypt_1 ...
Creating standalone_squidex_encrypt_1 ... done

...TempProjects\squidex-docker\standalone>docker ps
CONTAINER ID        IMAGE                                    COMMAND                  CREATED             STATUS              PORTS                                              NAMES
89be92c6adb7        jrcs/letsencrypt-nginx-proxy-companion   "/bin/bash /app/en..."   33 seconds ago      Up 32 seconds                                                          standalone_squidex_encrypt_1
3aece9b2f0fe        jwilder/nginx-proxy                      "/app/docker-entry..."   34 seconds ago      Up 33 seconds       0.0.0.0:81->81/tcp, 80/tcp, 0.0.0.0:443->443/tcp   standalone_squidex_proxy_1
254679061dc6        squidex/squidex:dev                      "dotnet Squidex.dll"     35 seconds ago      Up 34 seconds       80/tcp, 0.0.0.0:5000->5000/tcp                     standalone_squidex_squidex_1
bc8787d7ccae        mongo:latest                             "docker-entrypoint..."   58 seconds ago      Up 34 seconds       0.0.0.0:27017->27017/tcp                           standalone_squidex_mongo_1`
```
this do not give something 
> http://localhost:5000/

only 

``` 
This page isn’t working
localhost didn’t send any data.
ERR_EMPTY_RESPONSE
```

**Comment by SebastianStehle** [comment]
The docker-compose file is built to install Squidex in a production system where you can use port 80. Actually I don't know what you have changed ;)

What it does is to create a Nging Proxy that listens to Port 80 and 443. And a lets encrypt image that creates a cert for it for the domain that is defined in the .env file. 

If you don't need this stuff you can remove the nginx and lets encrypt containers. But nevertheless localhost:5000 should work but I never tested it. I tested this config by creating a VM on Vultr.

I think your config is wrong, because of this line: 5000->5000/tcp for your squidex container. Kestrel listens to port 80, therefore I map 5000 to 80 because Nginx needs to listen to port 80.
Basically it is 80 -> Nginx -> 5000 -> Docker -> 80 -> Kestrel.
